base/linalg/dense.jl

```diff
@@ -112,6 +112,43 @@ true
 isposdef(A::AbstractMatrix) = ishermitian(A) && isposdef(cholfact(Hermitian(A)))
 isposdef(x::Number) = imag(x)==0 && real(x) > 0
 
+"""
+    stride1(A) -> Int
+
+Returns the distance between successive array elements
+in units of element size.
+If the argument type is `Array`, the return value of `stride1` is always `1`.
+
+# Examples
+```jldoctest
+julia> A = [1,2,3,4,5,6,7,8,9,10]
```

Comment:
I suggest a smaller number of entries in `A` since the output is quite lengthy. Perhaps
````
```jldoctest
julia> A = [1, 2, 3, 4]
4-element Array{Int64,1}:
 1
 2
 3
 4

julia> Base.LinAlg.stride1(A)
1

julia> B = view(A, 2:2:4)
2-element view(::Array{Int64,1}, 2:2:4) with eltype Int64:
 2
 4

julia> Base.LinAlg.stride1(B)
2
```
````


Alternatively you could suppress the output with `;`.

Reply:
Thank you for your comment.
I used  a smaller number of entries in `A`.